notification-api-service/src/main/java/com/smarsh/notificationservice/api/util/PathUtil.java

```diff
@@ -12,19 +12,20 @@ private PathUtil() {
     }
 
     public static String resolvePath(TemplateXmlModel template) {
-        String path;
+        String path = null;
+        path.toString();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The return value of "toString" must be used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev-sonarqube.services.aws/coding_rules#rule_key=squid%3AS2201)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "path" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev-sonarqube.services.aws/coding_rules#rule_key=squid%3AS2259)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "path" is already a string, there's no need to call "toString()" on it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev-sonarqube.services.aws/coding_rules#rule_key=squid%3AS1858)